assets/app/views/attachPersistentVolumeClaim.html

```diff
@@ -0,0 +1,183 @@
+<div class="content">
+  <div class="container">
+    <div class="col-md-12">
+      <breadcrumbs breadcrumbs="breadcrumbs"></breadcrumbs>
+      <alerts alerts="alerts"></alerts>
+      <div class="row">
+        <div class="create-route-icon col-md-2 gutter-top hidden-sm hidden-xs">
+          <span class="fa fa-hdd-o icon-xl"></span>
+        </div>
+        <div class="col-md-8">
+          <h1>Attach Storage</h1>
+          <div>
+            <span class="help-block">
+              Attach an existing persistent volume claim to the pod template of a 
+              <span ng-if="attach.deploymentConfig">deployment config.</span>
+              <span ng-if="attach.deployment">deployment.</span>
+            </span>
+          </div>
+          <div ng-show="!pvcs || !(attach.deploymentConfig || attach.deployment)">Loading...</div>
+          <div ng-show="pvcs && (attach.deploymentConfig || attach.deployment)">
+            <form name="attachPersistentVolumeClaimForm">
+              <fieldset ng-disabled="disableInputs">
+                <!-- attaching to a deployment config -->
+                <div class="form-group" ng-if="attach.deploymentConfig">
+                  <label for="deployment-config-name" class="required">Deployment config</label>
+                  <input
```

Comment:
We hid the service input and updated the description at the top of the create route page to say, "Create a route to expose service {{serviceName}}."